### PR TITLE
Add SafeMath as npm dependency

### PR DESCRIPTION
**Pull request by schemar**
References https://github.com/OpenST/mosaic-contracts/issues/712

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "eslint-plugin-import": "2.14.0",
     "ganache-cli": "6.1.8",
     "mocha": "5.2.0",
+    "openzeppelin-solidity": "2.1.1",
```
Comment (deepesh-kn) [inline package.json:49]: Why not version 2.2.0 ?

**Review by deepesh-kn — COMMENTED** [review]
LGTM 👍 

Should `test/lib/safe_math.js` be deleted?

In general, should we include the tests for the external contracts? 
If tests are there then any breaking changes in the external contracts can be identified.

**Comment by schemar** [comment]
Thanks @deepesh-kn 👍

2.1.1 for the same reason as here: https://github.com/OpenST/mosaic-contracts/pull/728#issuecomment-490482368

Regarding testing external contracts, then I opt against it. In my opinion it is the responsibility of the maintainer of the dependency to test it. Otherwise we would need to test all our dependencies and all their dependencies 😱
OpenZeppelin has tests in place: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/test/math/SafeMath.test.js

As long as everyone sticks to SemVer and we use lock files/pinned version numbers, a breaking change _should_ not occur 🧐

What do you think?

**Comment by deepesh-kn** [comment]
> Thanks @deepesh-kn 👍
> 
> 2.1.1 for the same reason as here: [OpenST/mosaic-contracts#728 (comment)](https://github.com/OpenST/mosaic-contracts/pull/728#issuecomment-490482368)

👍 

> 
> Regarding testing external contracts, then I opt against it. In my opinion it is the responsibility of the maintainer of the dependency to test it. Otherwise we would need to test all our dependencies and all their dependencies 😱
> OpenZeppelin has tests in place: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/test/math/SafeMath.test.js
> 
> As long as everyone sticks to SemVer and we use lock files/pinned version numbers, a breaking change _should_ not occur 🧐
> 
> What do you think?


Make sense. I am fine with it.

**Review by deepesh-kn — APPROVED** [review]
LGTM 👍

**Pull request merged**